conanfile.py

```diff
@@ -62,7 +62,10 @@ def build_requirements(self):
             elif tools.os_info.is_linux and tools.os_info.linux_distro not in ["arch", "manjaro"]:
                 pack_names = ["libxcb-devel", "libX11-devel", "glibc-devel"]
                 if self.options.opengl == "desktop":
-                    pack_names.append("mesa-libGL-devel")
+                    if (tools.os_info.linux_distro == "opensuse-leap"):
```

Comment:
nitpicking: shouldn't it be `if tools.os_info.linux_distro.startswith("opensuse"):` ?

Reply:
As I only tested it on leap, I do not know if the package name is different on the other opensuse distros. So I just changed it for leap here. I will try to find some machines with non-leap on it.

Reply:
http://rpm.pbone.net/index.php3?stat=3&search=mesa-libGL-devel shows that it has always been with a capital M, so I took the liberty to modify your commit and push it directly

Reply:
great, thank you very much!